arduino-ide-extension/src/browser/contributions/edit-contributions.ts

```diff
@@ -141,6 +141,11 @@ ${value}
       label: nls.localize('arduino/editor/decreaseIndent', 'Decrease Indent'),
       order: '2',
     });
+    registry.registerMenuAction(ArduinoMenus.EDIT__CODE_CONTROL_GROUP, {
+      commandId: EditContributions.Commands.AUTO_FORMAT.id,
+      label: nls.localize('arduino/editor/autoFormat', 'Auto Format'), // XXX: The Java IDE uses `Use Selection For Find`.
```

Comment:
```suggestion
      label: nls.localize('arduino/editor/autoFormat', 'Auto Format'),
```
I could be wrong, but I suspect this comment was a copy/paste artifact that originated from here (where it was relevant):

https://github.com/arduino/arduino-ide/blob/6b32e793d05fcc4dc72ad59dc353ab72741529d8/arduino-ide-extension/src/browser/contributions/edit-contributions.ts#L188-L195

To there (where it was not relevant):

https://github.com/arduino/arduino-ide/blob/6b32e793d05fcc4dc72ad59dc353ab72741529d8/arduino-ide-extension/src/browser/contributions/edit-contributions.ts#L198-L202

If so, it is also not relevant here.

Reply:
Maybe you're right. 

For restoring this menu item I did the copy/paste directly from main:

https://github.com/arduino/arduino-ide/blob/8a1cabd2bcfcba822eca60a534058a2fe426234d/arduino-ide-extension/src/browser/contributions/edit-contributions.ts#L193-L198

Since it is not relevant I remove it. Thanks @per1234.